easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023b.yml

```diff
@@ -57,3 +57,13 @@ easyconfigs:
   - Qt5-5.15.13-GCCcore-13.2.0.eb:
       options:
         from-pr: 20201
+  - SuperLU_DIST-8.1.2-foss-2023a.eb:
+      options:
+        from-pr: 20162
+  - PETSc-3.20.3-foss-2023a.eb:
+      options:
+        include-easyblocks-from-pr: 3086
+        from-pr: 19686
+  - MODFLOW-6.4.4-foss-2023a.eb:
+      options:
+        from-pr: 20142
```

Comment:
Looks like you've added this to the wrong easystack file (2023b instead of 2023a)?

Reply:
Ugh... Good catch.

It doesn't matter too much in practice, but we should definitely try to avoid this slipping through somehow.
Not sure how we can easily catch this in CI...

Reply:
The good news is that this doesn't really affect the builds that were done, so this is ready to deploy?

Reply:
Fixed, but good point. Very sharp that you catch this, but very easy to miss. A CI check would be great...

Reply:
I agree, it doesn't affect the builds. They can be deployed without rebuilding.